hatch.toml

```diff
@@ -0,0 +1,47 @@
+[build.targets.wheel]
+only-include = ["pydra/tasks/freesurfer"]
+
```

Comment:
> Any particular reason for preferring `hatch.toml` to `pyproject.toml` for this?

I think you're right, this should probably go to `pyproject.toml`.

> Or for the environments?

I blindly followed the setup used by [hatch](https://github.com/pypa/hatch) itself tbh 😅 

I personally like the idea of having `hatch` environments, dependencies and scripts in a separate config file, and keep `pyproject.toml` for project metadata and tooling config.

Reply:
Sounds reasonable. I've split `pyproject.toml` and `tox.ini` in the past, though that's partially because tox's pyproject support is just "paste tox.ini inside triple-quotes".